mlir/include/mlir/Dialect/Rock/IR/RockAttrDefs.td

```diff
@@ -173,8 +177,7 @@ def Rock_TransformAttr : Rock_Attr<"Transform"> {
 }
 
 /// TransformsAttr
-def Rock_TransformMapAttr : Rock_Attr<"TransformMap"> {
-    let mnemonic = "transform_map";
+def Rock_TransformMapAttr : Rock_Attr<"TransformMap", "transform_map"> {
```

Comment:
Strong object: these attributes aren't the sort of discardable/dialect-wide attributes your setup is meant for

Reply:
They aren't using the new infra. I just moved the mnemonic setting to Rock_Attr.

Reply:
Yeah, fair enough